Azure/sonic-utilities
# Add support for port mirroring CLIs

Signed-off-by: Rupesh Kumar <rupesh-k.kumar@broadcom.com>

**- What I did**

Port mirroring CLI implementation 
HLD @ https://github.com/Azure/SONiC/pull/58

**- How I did it**
Enhance existing mirror_session command to support port mirroring.

**- How to verify it**
Create ERSPAN/SPAN mirror session and verify show mirror_session.

**- Previous command output (if the output of a command-line utility has changed)**

config mirror_session add everflow0 10.1.1.1 12.1.1.1 10 10
root@sonic:/home/admin# show mirror_session
Name       Status    SRC IP    DST IP      GRE    DSCP    TTL  Queue    Policer    Monitor Port    SRC Port    Direction
---------  --------  --------  --------  -----  ------  -----  -------  ---------  --------------  ----------  -----------
everflow0  active  10.1.1.1  12.1.1.1             10     10

**- New command output (if the output of a command-line utility has changed)**

config mirror_session add span sess1 Ethernet44 Ethernet40 rx

root@sonic:/home/admin# show mirror_session
ERSPAN Sessions
Name       Status    SRC IP    DST IP      GRE    DSCP    TTL  Queue    Policer    Monitor Port    SRC Port    Direction
---------  --------  --------  --------  -----  ------  -----  -------  ---------  --------------  ----------  -----------
everflow0  active  10.1.1.1  12.1.1.1      0      10     10

SPAN Sessions
Name    Status    DST Port    SRC Port    Direction    Queue    Policer
------  --------  ----------  ----------  -----------  -------  ---------
sess1    active     Ethernet44  Ethernet40  rx
